src/OpenSearch.OpenSearch.Managed/ConsoleWriters/LineOutParser.cs

```diff
@@ -119,9 +119,7 @@ private LineOutParser(string shortNamePrefix, string fullNamePrefix, string secu
 
 		private bool TryGetStartedConfirmation(string section, string message)
 		{
-
-			var inNodeSection = _securityPluginRegex.IsMatch(section);
```

Comment:
Can you remove `_securityPluginRegex`? It's not used any more.

Looks like you'll be able to remove `securityPluginName` parameter from `LineOutParser` as well.